src/power_shell_cache.erl

```diff
@@ -268,20 +268,21 @@ maybe_cover(Expanded, _File, Mod, Filename, Loaded, _Cover) ->
     FunMap = select_funs(Mod, Expanded),
     #module_data{
         hash = extract_hash(Mod, Loaded),
-        fun_map = maybe_onload(
-            %% compatibility behaviour: power_shell was skipping on_load
-            application:get_env(power_shell, skip_on_load, true),
-            Mod,
-            maps:get(on_load, FunMap, false),
-            FunMap),
+        fun_map = maybe_onload(Mod, FunMap),
         filename = Filename,
         mtime = filelib:last_modified(Filename)
     }.
 
-maybe_onload(false, Mod, FunName, FunMap) ->
-    ok = power_shell:eval(Mod, FunName, [], FunMap),
```

Comment:
Could you actually make the fix smaller, by doing this:
```
is_map_key(on_load, FunMap) andalso ok = power_shell:eval(Mod, FunName, [], FunMap)
```

This will reduce the change to just 1 line instead of 10

Reply:
I can, but the flow is odd, since you access the FunMap map twice:
1. extract the function with false as default value (actually doesn't matter what is default as lon as it doesn't crash)
2. afterwards check if FunName is in the map
It's backwards to me.